packages/client/src/components/ui/input/Input.tsx

```diff
@@ -0,0 +1,34 @@
+import { InputHTMLAttributes } from 'react'
+
+import style from './index.module.scss'
+
+export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  title?: string
+}
+
+export default function Input({
+  type = 'text',
+  name,
+  value,
+  onChange,
+  title,
+  id,
+  ...props
+}: InputProps) {
+  return (
+    <div className={style.input}>
+      <label className={style.label} htmlFor={id}>
+        {title}
+      </label>
+      <input
+        className={style.element}
```

Comment:
[Исправление]
Не передал `id` в `input`, не будет связи с `label`
```suggestion
        id={id}
        className={style.element}
```